php-practice/section4.php

```diff
@@ -17,7 +17,7 @@ class VideoGame {
   public $isMultiplayer; // Whether the game is a multiplayer game -- should be TRUE or FALSE
 
   // This is the contructor. Remember, this is the function that gets run whenever you create a new VideoGame object
-  public __construct($title, $publisher, $isMultiplayer) {
+  function __construct($title, $publisher, $isMultiplayer) {
```

Comment:
Good call on this -- it should have been `public function __construct`. (Although, if you don't specify the **visibility** [public/private] then it defaults to public for functions.)